emrichen/context.py

```diff
@@ -46,7 +45,7 @@ def add_variables(self, *variable_sources, **override_variables):
         """
         for variables in variable_sources:
             if not isinstance(variables, Mapping):
-                variables = yaml.load(variables, RichLoader)
+                variables = parse(variables, 'yaml')[0]
```

Comment:
Didn't think of it before, but if there are multiple YAML documents in a variable file, we could just pick them all up. I'll add it.